Tools/SwiftLint.md

```diff
@@ -16,10 +16,16 @@ If you have already installed SwiftLint, you can update it to the latest version
 If you want to integrate SwiftLint to Xcode, add the following script to your target:
 
 ```bash
+if test -d "/opt/homebrew/bin/"; then
+  PATH="/opt/homebrew/bin/:${PATH}"
+fi
+
+export PATH
```

Comment:
You should add this check only if a developer installed SwiftLint via Homebrew. 
Also, I would add two checks instead of one;

* If a user is on M1 Mac
* If a user has Homebrew installed

The reason for that is because that happens only in the M series. Check for M1 is available on SwiftLint's `README.md`;

```shell
if [[ "$(uname -m)" == arm64 ]]; then
    export PATH="/opt/homebrew/bin:$PATH"
fi
```

Reply:
I was going with the reasoning that:

1. We all use Apple Silicon now
2. The Homebrew installation is part of our handbook

so taking that into account, that's the most often used approach, so anything other than that approach should be added as a separate note in my opinion, if at all.

Reply:
I love the good argument and I agree with you 💪